README.md

```diff
@@ -1,25 +1,41 @@
 # Kubernetes Metrics Server
 
-Metrics server is one of the components in core metrics pipeline described in [Kubernetes monitoring architecture].
-Metrics server is responsible for collecting resource metrics from kubelets and exposing them in Kubernetes Apiserver
-through [Metrics API]. Main consumers of those metrics are `kubectl top`, [HPA] and [VPA]. Metric server stores only the
-latest values of metrics needed for core metrics pipeline (CPU, Memory) and is not responsible for forwarding metrics
-to third-party destinations.
+Metrics Server is a simple, scalable, resource efficient source of container resource metrics needed for Kubernetes
```

Comment:
It's definitely not [simple](http://harmful.cat-v.org/software/xml/soap/simple). 

```suggestion
Metrics Server is a scalable, efficient source of container resource metrics for Kubernetes
```